client/levels/menu.tscn

```diff
@@ -756,107 +826,172 @@ custom_minimum_size = Vector2(455, 500)
+  * Lead Level Designer
 
-Gloomreach
-[indent][i]Lead Unit Concept Artist[/i][/indent]
+[font_size=45]Gloomreach[/font_size]
+  * Lead Unit Concept Artist
 
-Zamza
-[indent][i]Spriter[/i][/indent]
+[font_size=45]Zamza[/font_size]
+  * Spriter
 
-Harmless
-[indent][i]Composer[/i][/indent]
+[font_size=45]Harmless[/font_size]
+  * Composer
 
 
-[center][b]Former[/b][/center]
+[font_size=54]Former[/font_size]
 
-Ghost
-[indent][i]Engine Developer[/i][/indent]
+[font_size=45]Ghost[/font_size]
+  * Engine Developer
 
-Darkbladercr
-[indent][i]Backend Developer[/i][/indent]
+[font_size=45]Darkbladercr[/font_size]
+  * Backend Developer
 
-seerfox
-[indent][i]CO Art Director[/i][/indent]
-[indent][i]UI Designer[/i][/indent]
+[font_size=45]seerfox[/font_size]
+  * CO Art Director
+  * UI Designer
 
-Sleepy Tech
-[indent][i]Spriter[/i][/indent]
+[font_size=45]Sleepy Tech[/font_size]
+  * Spriter
 
 
-[center][b]External[/b][/center]
+[font_size=54]External[/font_size]
 
-[url=https://www.kenney.nl/assets]Kenney Game Assets[/url]
-[indent][i]2D Art, Sound[/i][/indent]
+[font_size=45][url=https://www.kenney.nl/assets]Kenney Game Assets[/url][/font_size]
+  * 2D Art, Sound
 
-[url=https://opengameart.org/content/menu-music]mrpoly[/url]
-[indent][i]Music[/i][/indent]
+[font_size=45][url=https://opengameart.org/content/menu-music]mrpoly[/url][/font_size]
+  * Music
 
-[url=https://opengameart.org/content/awake-megawall-10]cynicmusic[/url]
-[indent][i]Music[/i][/indent]
+[font_size=45][url=https://opengameart.org/content/awake-megawall-10]cynicmusic[/url][/font_size]
+  * Music
 
-[url=https://opengameart.org/content/cyberpunk-moonlight-sonata]Joth[/url]
-[indent][i]Music[/i][/indent]
+[font_size=45][url=https://opengameart.org/content/cyberpunk-moonlight-sonata]Joth[/url][/font_size]
+  * Music
 
-[url=https://opengameart.org/content/4-chiptunes-adventure]SubspaceAudio[/url]
-[indent][i]Music[/i][/indent]
+[font_size=45][url=https://opengameart.org/content/4-chiptunes-adventure]SubspaceAudio[/url][/font_size]
+  * Music
 
-[url=https://akveo.github.io/eva-icons/]Eva Icons[/url]
-[indent][i]2D Art[/i][/indent]
+[font_size=45][url=https://akveo.github.io/eva-icons/]Eva Icons[/url][/font_size]
+  * 2D Art
 
 
-[center][b]Game Engine[/b][/center]
+[font_size=54]Game Engine[/font_size]
 
-[url=https://godotengine.org]Godot Engine[/url]
-[indent][i][url]https://godotengine.org/license[/url][/i][/indent]
+[font_size=45][url=https://godotengine.org]Godot Engine[/url][/font_size]
+  * [url=https://godotengine.org/license]licence[/url]
 
-[center][img]res://godot_icon.svg[/img][/center]
+[img]res://godot_icon.svg[/img]
 
 
-[url=https://github.com/Codename-Recon/Codename-Recon/blob/master/LICENSE_ASSETS.md]Asset License[/url]"
-script = SubResource("GDScript_xjdvj")
+[url=https://github.com/Codename-Recon/Codename-Recon/blob/master/LICENSE_ASSETS.md]Asset License[/url]
+"
+script = ExtResource("19_8a2oa")
+markdown_text = "# Credits
```

Comment:
with this aproach library, we get the credits by duplicated, as it saves the text variable of the richtext, is something I don't like it too much. 

A proposal will be, that we have a softlink (I don't know if git handles it), or similar stuff, and have the markdown as a file in the project (**included in the exports too**) and we simply load that on the menu start or similar. What do you think?

Reply:
Sounds good. You could place it under `client/assets/text/`. Keep in mind to update this field for every export variation to include Markdown files: 
![image](https://github.com/Codename-Recon/Codename-Recon/assets/1898454/35da6964-2333-40a7-8d11-f357dfc8c8af)